src/egon/data/datasets/scenario_capacities.py

```diff
@@ -846,17 +846,10 @@ def eGon100_capacities():
         )
 
 
-tasks = (create_table,)
+tasks = (create_table, insert_data_nep,)
 
 if "status2019" in egon.data.config.settings()["egon-data"]["--scenarios"]:
-    tasks = tasks + (insert_capacities_status2019, insert_data_nep)
```

Comment:
For the status2019 scenario the order of the tasks changed. Do you think that could lead to problems?

Reply:
Just in case I just checked and there should not be any problem related to the order.  Data written for the task __insert_data_net__ is not used for other tasks in the same dataset.